lutris/util/system.py

```diff
@@ -455,3 +455,18 @@ def is_running(process):
         if re.search(process, x):
             return True
     return False
+
+
+def check_ntfs(path):
+    """Returns True if the given path is located on a NTFS partition"""
+
+    out = None
+
+    df_cmd = find_executable('df')
+    if df_cmd and path:
+        out = subprocess.check_output(
+            [df_cmd, "-T", path]).decode("UTF-8")
+
+    if out and "fuseblk" in out:
```

Comment:
Correct me, but that just means that the path is mounted over the FUSE driver and does not tell you anything about the type of the filesystem.

Reply:
I did a bit of research on this and experimentation and found that df command reports ntfs as fuseblk, and that's why I use it, I'm not sure if it's 100% valid but I don't know if it's not either^^

Reply:
`fuseblk` is simply the indicator that it is mounted through the FUSE driver, which could be NTFS or some other filesystem. So it is not limited to NTFS. Something like [this might be more reliable](https://unix.stackexchange.com/a/402236/4106).

Reply:
that's what fuseblk stands for, fuseblk returned from df or stat command which both share the same code base http://git.savannah.gnu.org/gitweb/?p=coreutils.git;a=blob_plain;f=src/stat.c will report ntfs as fuseblk at least on recent kernels, after more digging it seems these gnu tools used to report it as ntfs, so I'll probably add or ntfs in the check

Reply:
even if stat.c has a ntfs check git.savannah.gnu.org/gitweb/?p=coreutils.git;a=blob;f=src/stat.c#l415, apparently on recent kernels this path is not hit, I got curious now wonder if someone can explain it ^^

Reply:
This `lsblk -no MOUNTPOINT,fstype` actually is the first tool I used that reports ntfs, I guess I'll just change to that, guess linux tools can do what gnu tools cant ^^